interbtc-spec/docs/source/spec/issue.rst

```diff
@@ -196,7 +196,8 @@ Specification
     * The Vault's ``toBeIssuedTokens`` MUST decrease by the deficit (``issue.amount - amountTransferred``).
     * The ``issue.fee`` MUST be updated to the amount transferred multiplied by the :ref:`issueFee`.
     * The ``issue.amount`` MUST be set to the amount transferred minus the updated ``issue.fee``.
-    * The Vault's free balance MUST increase by the ``griefingCollateral``.
+    * The requester's free balance MUST increase by ``(griefingCollateral * amountTransferred) / (issue.amount + issue.fee)``.
```

Comment:
At this point the ``issue.amount`` might already be updated depending if you read the post conditions in order. So just to avoid confusion, I'd suggest to move this above the `* The ``issue.fee`` MUST be upd....` part